compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Utils/AMDAIELogicalObjFifoSplittingUtils.h

```diff
@@ -38,9 +38,11 @@ LogicalResult splitLogicalObjectFifo(
     int64_t splitStride = 1);
 
 /// Split doubly strided operations on a source and target split dimension with
-/// the provided split factor. If no split factor is provided, the doubly
-/// strided operation will be split on the size of the dimension being split.
-LogicalResult splitDoublyStridedOp(
+/// the provided split factor which might get updated. On success, return the
+/// split factor to the caller, else return failure.
```

Comment:
Now that the `splittingFactor` is returned and expected to be used in the user, this is getting too coupled with the user function. We should extract the `gcd` function and update logic from this function and put it in the user instead.